peanut_gb.h

```diff
@@ -3502,6 +3916,20 @@ void gb_reset(struct gb_s *gb)
 		gb->hram_io[IO_LCDC] = 0x91;
 		gb->hram_io[IO_STAT] = 0x85;
 		gb->hram_io[IO_BANK] = 0x01;
+#if PEANUT_FULL_GBC_SUPPORT
+		if (gb->cgb.cgbMode)
+		{
+			gb->cpu_reg.a = 0x11;
+			gb->cpu_reg.f_bits.z = 1;
+			gb->cpu_reg.f_bits.n = 0;
+			gb->cpu_reg.f_bits.h = hdr_chk;
+			gb->cpu_reg.f_bits.c = hdr_chk;
+		}
+		if (gb->cgb.cgbMode) gb->cpu_reg.bc.reg = 0x0000;
+		if (gb->cgb.cgbMode) gb->cpu_reg.de.reg = 0x0008;
+		if (gb->cgb.cgbMode) gb->cpu_reg.hl.reg = 0x007C;
+		if (gb->cgb.cgbMode) gb->hram_io[IO_DIV] = 0xFF;
```

Comment:
These should be in the same if statement as above. No need for repeated checks of cgbMode.